jena-arq/src/main/java/org/apache/jena/riot/lang/rdfxml/rrx_stax_ev/ParserRDFXML_StAX_EV.java

```diff
@@ -638,9 +637,15 @@ private XMLEvent propertyElementProcess(Node subject, StartElement startElt, Cou
         }
 
         String parseTypeName = parseType;
-        if ( parseTypeName.equals(parseTypeLiteralAlt) ) {
-            Log.warn(SysRIOT.getLogger(), "Encountered rdf:parseType='literal'. Treated as rdf:parseType='literal'");
-            parseTypeName = "Literal";
+        switch( parseTypeName) {
+            case parseTypeLiteralAlt -> {
+                RDFXMLparseWarning("Encountered rdf:parseType='literal'. Treated as rdf:parseType='literal'", location);
```

Comment:
Should the second part of the error message be `Treated as rdf:parseType='Literal'`? Just checking because `rdf:parseType='literal'` appears twice in the message....

Reply:
Good catch!

Reply:
Looks good to me now, thanks!